### DISCUSSION TITLE
Specifying the language

### DISCUSSION THREAD

**Discussion by agharbeia** [ricklamers/gpt-code-ui#86] [Q&A]
Whenever I specify a programming language for the generated code, confusion ensues!

The model always seems to think that I requested "Python" as the programming language. It cites that "Python" is specified in "Directions".

I suspect some sort of prompt is either hard-coded, or somewhere configurable, that causes this?

If it is configurable, then where?

**Comment by ricklamers** [comment]
https://github.com/ricklamers/gpt-code-ui/blob/82dbd1dc0803c7682a34fcb6df70082e6f08ef6f/gpt_code_ui/webapp/main.py#L102

This is a pretty Python centric project, but in principle you could use a different Jupyter kernel and change the prompts like the one linked to above.
